src/main/java/org/meteordev/starscript/StandardLib.java

```diff
@@ -166,4 +173,47 @@ public static Value pad(Starscript ss, int argCount) {
 
         return Value.string(new String(padded));
     }
+
+    public static Value formatDateTime(Starscript ss, int argCount) {
+        if (argCount < 1 || argCount > 2) ss.error("formatTime(fmt, timezone) requires 1 to 2 arguments, got %d.", argCount);
+        try {
+            String timeZone = null;
+            if (argCount == 2) timeZone = ss.popString("Argument to formatTime(fmt, timezone) needs to be a string.");
+            String fmt = ss.popString("Argument to formatTime(fmt, timezone) needs to be a string.");
+            SimpleDateFormat formatter = new SimpleDateFormat(fmt);
+            if (timeZone != null && !timeZone.isEmpty()) formatter.setTimeZone(TimeZone.getTimeZone(timeZone));
+            return Value.string(formatter.format(new Date()));
+        }
+        catch (IllegalArgumentException e) {
+            ss.error(e.toString());
+        }
+        return Value.null_();
+    }
+
+    public static Value format(Starscript ss, int argCount) {
+        if (argCount < 1) ss.error("format(fmt, ...args) requires at least 1 argument, got %d.", argCount);
+        ArrayList<Object> args = new ArrayList<Object>();
+        for (int i = 1; i < argCount; i ++) {
+            Value v = ss.pop();
+            Object o = null;
+            switch (v.type) {
+                case Boolean: o = v.getBool(); break;
+                case Number: o = v.getNumber(); break;
+                case String: o = v.getString(); break;
+                case Function: o = v.getFunction(); break;
+                case Map: o = v.getMap(); break;
+                case Null:
```

Comment:
instead of initializing `Object o` as `null` and simply breaking out of the switch on `case Null`/`default`, it would be more readable if `o` wasn't initialized to a value, but `case Null`/`default` set it to `null`